docs/config.md

```diff
@@ -56,6 +54,25 @@ Depending on Node's mode, make sure you're using the correct extension and synta
 
 > Try to stick with the `.js` extension whenever possible.
 
+### Disable automatic handling of svelte config
+
+Use `configFile: false` to prevent vite-plugin-svelte from reading the config file or restarting the devserver when it changes.
```

Comment:
```suggestion
Use `configFile: false` to prevent `vite-plugin-svelte` from reading the config file or restarting the Vite dev server when it changes.
```